manifests/d/Datadog/dd-trace-dotnet/1.27.0/Datadog.dd-trace-dotnet.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: Datadog.dd-trace-dotnet
+PackageVersion: 1.27.0
+PackageLocale: en-US
+Publisher: Datadog, Inc.
+PublisherUrl: https://docs.datadoghq.com/
+PublisherSupportUrl: https://www.datadoghq.com/support/
+PrivacyUrl: https://www.datadoghq.com/legal/privacy/
+Author: Datadog
+PackageName: Datadog APM .NET Tracer
```

Comment:
```suggestion
PackageName: Datadog .NET Tracer 64-bit
```
![image](https://user-images.githubusercontent.com/11600822/120545454-9420bd00-c3e6-11eb-9e68-bf8facfbe83b.png)

Reply:
Thanks, I removed "APM" from the name to better match the msi product name, but we cannot add "64-bit". We actually have two msi installers (x64 and x86) with different `ProductName` (one says "64-bit" and the other "32-bit"). For now we're only listing the x64 installer, but the goal is to list both installers once that is supported by winget, so the top-level `PackageName` in the manifest cannot match the different `ProductName` in the two msi.

- winget `PackageName`: Datadog .NET Tracer
  - x64 msi `ProductName`: Datadog .NET Tracer 64-bit
  - x86 msi `ProductName`: Datadog .NET Tracer 32-bit

Reply:
That's fine :)